CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 3.1.0
+
+ * This is the first release that will be supplied as binary wheels (manylinux2010).
+ * Note that you need pip > 10.x to use manylinux2010 wheels.
+ * If you don't use the binary wheel and unicode is important to you; use casacore 3.1.1 
```

Comment:
`;` -> `,`

Reply:
I think the above 3 lines would be more readable as a normal paragraph rather than separate bullet points (because they all relate to a single topic).